src/views/admin/AdminLTCQuestView.vue

```diff
@@ -1,10 +1,138 @@
+        const [startDate, endDate] = taskDates.value;
+        const newTask = new Task(
+            tasksData.length,
+            tasksData.length,
+            taskTitle.value,
+            taskDescription.value,
+            taskInfo.value,
+            TASK_STATUS.UNDONE,
+            taskPoints.value,
+            ltcQuestStore__isSpecial.isSpecial,
+            2, // Placeholder creatorId
+            new Date().toISOString(),
+            ltcQuestStore__isTeam.isTeamMode ? selectedParticipants.value : [], // Teams
+            !ltcQuestStore__isTeam.isTeamMode ? selectedParticipants.value : [], // Users
+            "",
+            null,
+            taskPhotos.value,
+            startDate ? startDate.toISOString() : null,
+            endDate ? endDate.toISOString() : null,
+        );
+        tasksData.push(newTask);
+        dialog.value = false;
+        taskCreatedSnackbar.value = true;
+        resetForm();
+    }
+};
 </script>
 
 <template>
-    <v-switch v-model="ltcQuestStore.isPointsBarsAvailable" :label="$t('ltcQuest.showPointsBars')" color="primary">
+    <v-switch v-model="ltcQuestStore__pointsBars.isPointsBarsAvailable" :label="$t('ltcQuest.showPointsBars')"
+        color="primary">
     </v-switch>
-</template>
\ No newline at end of file
+    <v-container>
+        <v-btn color="primary" @click="dialog = true">
+            {{ $t("taskCreator.createTask") }}
+        </v-btn>
+        <v-dialog v-model="dialog" max-width="800px">
+            <v-card>
+                <v-card-title>
+                    <span class="headline">{{ $t("taskCreator.creatorHeadline") }}</span>
+                </v-card-title>
+                <v-card-text>
+                    <v-form ref="form" v-model="isFormValid">
+                        <GameModeSelection />
+                        <ParticipantsSelection />
+
+                        <v-text-field :label="$t('taskCreator.taskTitle')" v-model="taskTitle" required
+                            :rules="[(v: string) => !!v || $t('taskCreator.requirementsValidation')]"></v-text-field>
+
+                        <v-textarea :label="$t('taskCreator.taskDescription')" v-model="taskDescription" required
+                            :rules="[(v: string) => !!v || $t('taskCreator.requirementsValidation')]"></v-textarea>
+                        <v-textarea :label="$t('taskCreator.additionalInfo')" v-model="taskInfo"></v-textarea>
+
+                        <v-text-field :label="$t('taskCreator.taskPoints')" v-model="taskPoints" type="number" min="1"
+                            :rules="[
+                                (v: number) => v >= 1 || $t('taskCreator.taskPointsValidation')
+                            ]"></v-text-field>
+
+                        <TaskTypeSelection v-model="taskType" />
+
+                        <HubDatePicker v-model="taskDates" :label="$t('taskCreator.taskTime')" :enableTimePicker="true"
+                            :min-date="minDate" />
```

Comment:
```suggestion
                        <HubDatePicker v-model="taskDates" :label="$t('taskCreator.taskTime')" enableTimePicker
                            :min-date="minDate" />
```
Mozna tak skracac. Tylko ze skoro Dajesz mozliwosc wyboru co do godziny to w innych miejscach w aplikacji tez powinnismy wyswietlac z godzinami a chyba na ten moment nie wyswietlamy wcale albo tylko dni.

Reply:
Myślałem, żeby dodać godziny w razie jakiś krótkich zadań. To usunąć to czy zmodyfikować HubDatePicker i dodać taką funkcję?

Reply:
Dobry pomysł z godzinowymi eventami/taskami. Tylko trzeba o tym pamiętać w pozostalych miejscach w aplikacji.